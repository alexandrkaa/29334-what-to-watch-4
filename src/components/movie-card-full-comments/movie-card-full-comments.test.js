import React from 'react';
import renderer from 'react-test-renderer';
import MovieCardFullComments from './movie-card-full-comments.jsx';

const comments = [
  {
    id: 1,
    author: {
      id: 17,
      name: `Emely`
    },
    rating: 2.9,
    comment: `Poised to be an instant classic, almost everything about this film is phenomenal - the acting, the cinematography, the discography, etc.`,
    date: `2020-07-15T12:25:15.535Z`
  },
  {
    id: 2,
    author: {
      id: 18,
      name: `Emely2`
    },
    rating: 2.9,
    comment: `comment text`,
    date: `2020-01-15T12:25:15.535Z`
  }
];

it(`<MovieCardFullComments /> should match snapshot`, () => {
  const tree = renderer
    .create(
        <MovieCardFullComments
          comments={comments}
        />
    )
    .toJSON();
  expect(tree).toMatchSnapshot();
});
