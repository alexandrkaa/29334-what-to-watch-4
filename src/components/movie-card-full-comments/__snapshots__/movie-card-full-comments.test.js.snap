// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<MovieCardFullComments /> should match snapshot 1`] = `
<div
  className="movie-card__reviews movie-card__row"
>
  <div
    className="movie-card__reviews-col"
  >
    <div
      className="review"
    >
      <blockquote
        className="review__quote"
      >
        <p
          className="review__text"
        />
        <footer
          className="review__details"
        >
          <cite
            className="review__author"
          >
            Emely
          </cite>
          <time
            className="review__date"
            dateTime="2020-07-15T12:25:15.535Z"
          >
            2020-07-15T12:25:15.535Z
          </time>
        </footer>
      </blockquote>
      <div
        className="review__rating"
      >
        2.9
      </div>
    </div>
  </div>
  <div
    className="movie-card__reviews-col"
  >
    <div
      className="review"
    >
      <blockquote
        className="review__quote"
      >
        <p
          className="review__text"
        />
        <footer
          className="review__details"
        >
          <cite
            className="review__author"
          >
            Emely2
          </cite>
          <time
            className="review__date"
            dateTime="2020-01-15T12:25:15.535Z"
          >
            2020-01-15T12:25:15.535Z
          </time>
        </footer>
      </blockquote>
      <div
        className="review__rating"
      >
        2.9
      </div>
    </div>
  </div>
</div>
`;
